index.js

```diff
@@ -21,24 +21,24 @@ function buildTargets(options) {
   return assign({}, defaultTargets, options.additionalTargets);
 }
 
-module.exports = function buildAirbnbPreset(context, options) {
+module.exports = function buildAirbnbPreset(api, options) {
   var transpileTargets = (options && options.targets) || buildTargets(options || {});
 
   var debug = (options && typeof options.debug === 'boolean') ? !!options.debug : false;
 
   return {
     presets: [
-      require('babel-preset-env').default(null, {
+      require('@babel/preset-env').default(api, {
```

Comment:
There's no reason to call this function manually, and in fact can cause bugs since `api` is an API for use in a specific preset only. It should be

```
presets: [
  [require('@babel/preset-env'), {
```
so that Babel is responsible for instantiating the preset.

Reply:
Fixed